internal/node/services.go

```diff
@@ -140,3 +142,30 @@ func newHttpService(c config.NodeConfig) services.HttpService {
 		Handler: handler,
 	}
 }
+
+func newPostgraphileService(c config.NodeConfig, workDir string) services.CommandService {
+	var s services.CommandService
+	s.Name = "postgraphile"
+	s.HealthcheckPort = getPort(c, portOffsetPostgraphile)
+	s.Path = "postgraphile"
+	s.Args = append(s.Args, "--retry-on-init-fail")
+	s.Args = append(s.Args, "--dynamic-json")
+	s.Args = append(s.Args, "--no-setof-functions-contain-nulls")
+	s.Args = append(s.Args, "--no-ignore-rbac")
+	s.Args = append(s.Args, "--disable-graphiql")
```

Comment:
By default, the current node enables the GraphQL browser interface. Why disable it?